### PR TITLE
Ko makes a python sim only pympsim dir

### PR DESCRIPTION
**Pull request by GANDAMU**
[It looks like you deleted a lot of the C++ simulator files and committed them.](https://github.com/jdblair/MegaPrayer-oscled/compare/master...KO_makes_a_python_sim?expand=1#diff-3aeaf994ecee7df75daa9136226e0412) Whether it was on purpose or by accident, I think we should keep those old files around.

Things got wonky when I told you to do the force push, so even though this branch is technically behind in a lot of places, the timestamps on the files make Github think you're ahead.

What I've done is check your pympsim directory files into the master branch, and made this new branch. It unfortunately doesn't have the history of the other branch.

Check this branch out, see if it has all the things you expect it to have from your own branch. And if it does, we can merge to master and you can move on to your next piece of the project.

In the end though, this is still your branch so you get the final say. The reason this is a pull request is that I want to give you the opportunity to reject it.

### CODE REVIEW

File: python/testMP.py [hunk 1]
```diff
@@ -9,19 +10,21 @@
     parser.add_argument("--port", default=5005, help="OSC server port")
     args = parser.parse_args()
 
-    r = rosary.Rosary(args.ip, args.port)
+    d = dispatcher.Dispatcher()
+    r = rosary.Rosary(args.ip, args.port, d)
```
Comment (GANDAMU) [inline python/testMP.py:14]: If you remember, this is the cause of that error that prompted me to say that my stuff wasn't playing nicely with your stuff, aka
`* ROSARY PATHS *
/rosary/pause
Traceback (most recent call last):
  File "testMP.py", line 12, in <module>
    r = rosary.Rosary(args.ip, args.port)
  File "/home/yunfan/jdb/MegaPrayer-oscled-alt/python/mp/rosary.py", line 116, in __init__
    self.register_with_dispatcher()
  File "/home/yunfan/jdb/MegaPrayer-oscled-alt/python/mp/rosary.py", line 408, in register_with_dispatcher
    self.dispatcher.map(osc_path,
AttributeError: 'NoneType' object has no attribute 'map'`

I'm still considering maybe just initializing a dispatcher.Dispatcher() object in the rosary if it's not passed...BUT I don't think this should affect you moving forward? So I might just leave it.

In other words, if this is the last time in your life you have to initialize a Rosary object I might just take the lazy route.

File: python/testMP.py [hunk 2]
```diff
@@ -9,19 +10,21 @@
...
 
     # create a bounce effect on all beads
-    r.add_effect('bounce', r.set_registry['All'])
+    #r.add_effect('bounce', r.set_registry['All'])
+    r.add_effect('bounce', 'all')
```
Comment (GANDAMU) [inline python/testMP.py:18]: Still not sure if I like that I replaced `r.set_registry['All']` with 'all', but again I'm just trying to get through this right now.

File: python/testMP.py [hunk 3]
```diff
@@ -9,19 +10,21 @@
...
 
     # change color to red
-    r.effect(1).color.set(r.color_registry['Red'])
+    r.effect(1).color.set(r.color_registry['red'])
 
     # add another throb on the odd beads
-    r.add_effect('throb', r.set_registry['Odd_Ring'], color=r.color_registry['Blue'])
+    r.add_effect('throb', 'odd_ring', color_name_or_r='blue')
```
Comment (GANDAMU) [inline python/testMP.py:24]: I think we all agree by now that `color_name_or_r` was a terrible idea. I'll fix it later.

**Review by KellyOgilvie — APPROVED** [review]
